packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -225,31 +230,34 @@ export async function batchInstallFontFaces( fontFamilyId, fontFacesData ) {
 /*
  * Downloads a font face asset from a URL to the client and returns a File object.
  */
-export async function downloadFontFaceAsset( url ) {
-	return fetch( new Request( url ) )
-		.then( ( response ) => {
-			if ( ! response.ok ) {
-				throw new Error(
-					`Error downloading font face asset from ${ url }. Server responded with status: ${ response.status }`
-				);
-			}
-			return response.blob();
-		} )
-		.then( ( blob ) => {
-			const filename = url.split( '/' ).pop();
-			const file = new File( [ blob ], filename, {
-				type: blob.type,
-			} );
-			return file;
+export async function downloadFontFaceAssets( src ) {
+	//src could be a single string or a collection of strings.
+	//here, just make sure it's an array
+	src = Array.isArray( src ) ? src : [ src ];
+
+	const files = await Promise.all(
+		src.map( async ( url ) => {
+			return await fetch( new Request( url ) )
```

Comment:
I don't think `await` is needed here, `fetch` returns a Promise, and we're already awaiting all of the Promises above.

Reply:
If we don't use `await` then `files` === a Promise rather than what the promise resolves to (which is an array of what is returned  from the `then()` which are blobs).